src/redux/configureStore.js

```diff
@@ -8,5 +8,7 @@ const rootReducer = combineReducers({
 });
 
 const store = configureStore({ reducer: rootReducer });
+// console.log(store);
+// console.log(store.getState());
```

Comment:
- [ ] Kindly consider removing old code, instead of commenting on it, this will make your code clean and easy to maintain

Reply:
Thanks, @Ismailco, fixed.